src/pgsql/bin/functions/create_symlinks

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+[ "$SYMLINKS" = "" ] && exit 0
+
+for LINK in $(echo $SYMLINKS | tr ',' ' ')
+do
+  TARGET=$(echo $LINK | cut -d ':' -f1)
+  echo "Target = $TARGET"
+  SOURCE=$(echo $LINK | cut -d ':' -f2)
+  echo "Source = $SOURCE"
```

Comment:
can be done in one line
https://gist.github.com/paunin/a7fad57f78c7dd15a63182c224126cc8#file-config-changer-sh-L15